src/platforms/ios/Podfile

```diff
@@ -1 +1,2 @@
-pod 'ISMessages', :git => 'https://github.com/EddyVerbruggen/ISMessages.git', :commit => '5763224d405256a116def1d405b43f7d30983f4d'
```

Comment:
I removed the hardcoded commit since I figure that we'd want the latest commit in your fork anyway.

Reply:
Good point